addons/l10n_fr_fec/i18n/l10n_fr_fec.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 14.0\n"
```

Comment:
?

Reply:
!

Reply:
Why are you changing the translation template?

Reply:
My directives concerning .pot were out-of-date. Will remove them